server/sonar-server/src/main/java/org/sonar/server/es/EsClient.java

```diff
@@ -193,6 +202,17 @@ public Client nativeClient() {
     return nativeClient;
   }
 
+  /**
+   * Checks whether there is any document in any mentioned type.
+   */
+  public boolean isEmpty(IndexTypeId... indexType) {
+    return count(indexType) <= 0;
+  }
+
+  public long count(IndexTypeId... indexType) {
```

Comment:
I would like to have the method public, for future use cases. I already have seen some pieces of code, that could use this method, when we migrate to es 5.